frontend/memegod/public/manifest.json

```diff
@@ -0,0 +1,25 @@
+{
+  "short_name": "React App",
```

Comment:
Please change the manifest.json accordingly

Reply:
Done